hieradata_aws/class/staging/backend.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+hosts::migration::hosts:
+  publishing-api-1:
+    ip: '10.2.3.45'
```

Comment:
I think this needs to be 10.3.3.x. Should it also include `publishing-api-3`? If so, that's 10.3.3.47.

Reply:
yes, we are checking now, maybe the list needs to be expanded